util-browser.js

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+// browser TextDecoder/TextEncoder
+/* eslint-env browser */
+const {TextDecoder, TextEncoder} = self;
+
+export function createJwsSigningInput({encodedHeader, encodedPayload}) {
+  return new TextEncoder().encode(encodedHeader + '.' + encodedPayload);
```

Comment:
Can we just create local TextEncoder and TextDecoder instances once and reuse them?

Reply:
Resolved.